docs/documentation/changelog.md

```diff
@@ -78,11 +75,11 @@ denote the various action types:![add](../media/img/add.jpg)=add,
 <img alt="update" src="../media/img/update.jpg" />=update
 ***
 
-{% include changelog/42.1.1-release.md %}
-
-{% include changelog/42.1.0-release.md %}
-
-{% include changelog/42.0.0-release.md %}
+{% for post in site.categories.new_release %}
+<a name="version_{{ post.version }}"></a>
+## Version {{ post.version }} ({{ post.date | date: "%Y-%d-%m"}})
+{{ post }}
+{% endfor %}
```

Comment:
This section fails with Jekyll 2.2, if that version is still being used in the web server, it probably looks bad there too.
The output is something like this:
```
{"title"=>"PostgreSQL JDBC Driver 42.1.2 Released", "date"=>2017-07-11 09:53:32 -0600, "categories"=>["newrelease"], "version"=>"42.1.2", "url"=>"/newrelease/2017/07/11/42.1.2-release.html", "dir"=>"/newrelease/2017/07/11", "id"=>"/newrelease/2017/07/11/42.1.2-release", "next"=>nil, "previous"=>, "tags"=>[], "path"=>"posts/2017-07-11-42.1.2-release.md", "content"=>"

Notable changes
\n\n

    \n
    Better logic for returning keyword detection. Previously, pgjdbc could be defeated by column names that contain returning, so pgjdbc failed to "return generated keys" as it considered statement as already having returning keyword PR#824 201daf1d
    \n
    Replication API: fix issue #834 setting statusIntervalUpdate causes high CPU load PR#835 59236b74
    \n
    perf: use server-prepared statements for batch inserts when prepareThreshold>0. Note: this enables batch to use server-prepared from the first executeBatch() execution (previously it waited for prepareThreshold executeBatch() calls) abc3d9d7\n<!--more-->
    \n

\n\n
```

In Jekyll 3.4 it looks good, but if the server is not upgraded yet to Debian 9 (Jekyll 3.1.6) this could cause trouble.